index.html

```diff
@@ -594,52 +596,57 @@
 S_\mathrm{subsurface} = \mathrm{Slab}(f_\mathrm{dielectric}, V^\infty_\mathrm{subsurface}) \ .
 \end{equation}
 
-As for the [glossy-diffuse](index.html#model/dielectricbase/glossy-diffuse) slab and the [translucent-base](index.html#model/dielectricbase/translucentbase), the subsurface is bounded by a dielectric interface with BSDF $f_\mathrm{dielectric}$, which generates the primary specular lobe parametrized via the "specular" parameters as described in the Dielectric base section.
+As for the [glossy-diffuse](index.html#model/dielectricbase/glossy-diffuse) slab and the [translucent-base](index.html#model/dielectricbase/translucentbase), the subsurface is bounded by a dielectric interface with BSDF $f_\mathrm{dielectric}$, which generates the primary specular lobe parametrized via the "specular" parameters as described in the Dielectric base section. Superimposed on this is the reflection generated by scattering in the subsurface medium. In this case the subsurface medium $V^\infty_\mathrm{subsurface}$ is given a parametrization which is particularly convenient for controlling the volumetric effect of dense subsurface scattering:
 
-Superimposed on this is the reflection generated by scattering in the subsurface medium. In this case the subsurface medium $V^\infty_\mathrm{subsurface}$ is given a parametrization which is particularly convenient for controlling the volumetric effect of dense subsurface scattering:
+   - **`subsurface_radius`** * **`subsurface_radius_scale`**: the _mean free path_ (MFP) per RGB channel, $\mathbf{r}$, i.e. the typical distance that light travels in the medium before absorption or scattering. This thus controls the apparent density of the medium. In the limit of zero MFP, the medium tends towards infinite density, and approaches the look of an opaque diffuse surface. Being a length, **`subsurface_radius`** can be any value greater than or equal to zero. For convenience, we make the soft range $[0, 1]$, thus covering common cases where the MFP is typically much lower than the scene length units. The **`subsurface_radius_scale`** controls the color channel dependence of the MFP, and thus this color is visible in the light transmitted through thinner regions of the subsurface volume.
+   - **`subsurface_color`**: the observed RGB reflection albedo color taking into account all orders of multiple scattering, $\mathbf{C}$ (where the sense in which this parametrizes the observed color is discussed in detail below).
+   - **`subsurface_anisotropy`**: the medium's phase function, parametrized by the scalar anisotropy $g \in [-1, 1]$.
```

Comment:
Do we intentionally want to support $g \in [-1, 1]$ or just $g \in (-1, 1)$? I suppose the implementation is responsible for either clamping the input $g$ or having a special case for perfect forward and backward scattering.

Reply:
I think in practice one needs to clamp, as various formulas blow up at the limits (and also the rendering gets pathological well before reaching the bounds). It might be convenient though from an artist standpoint to be able to dial all the way to the bounds, and still get something sensible, relying on an internal clamp. It seems like a good idea to explicitly mention that implementations may internally clamp the anisotropy to avoid such pathologies.

Reply:
That makes sense.

> It seems like a good idea to explicitly mention that implementations may internally clamp the anisotropy to avoid such pathologies.

That would be nice (assuming it's not obvious).

Specular reflection has similar issues when the roughness approaches zero. Not sure if we need to mention that or if it's implied that implementations will have to handle certain mathematical and numerical corner cases.

Reply:
Suggested clarification:
```suggestion
   - **`subsurface_anisotropy`**: the medium's phase function, parametrized by the scalar anisotropy $g \in [-1, 1]$. In practice implementations may want to internally clamp the anisotropy as the phase function becomes ill-defined and the rendering unstable near the -1, 1 limits.
```

Reply:
That clarification sounds good to me.